templates/hackmd/0/docker-compose.yml

```diff
@@ -0,0 +1,43 @@
+hackmdPostgres:
+  image: postgres
+  environment:
+    POSTGRES_USER: hackmd
+    POSTGRES_PASSWORD: hackmdpass
+    POSTGRES_DB: hackmd
+
+hackmd:
+  image: hackmdio/hackmd:0.4.5
+  environment:
+    POSTGRES_USER: hackmd
+    POSTGRES_PASSWORD: hackmdpass
+    NODE_ENV: "${NODE_ENV}"
+    DEBUG: "${DEBUG}"
+    HMD_DOMAIN: "${HMD_DOMAIN}"
+    HMD_URL_PATH: "${HMD_URL_PATH}"
+    HMD_PORT: "${HMD_PORT}"
```

Comment:
@elboletaire If you don't want to expose a node port for the template then you can add a README file explaining how to use the template and in it you can explain that the user can open a port manually for the container or add a separate load balancer.